### ISSUE TITLE
automate publish docs

### ISSUE THREAD

**Issue by gjf7** [neovim/node-client#428]
The docs website is currently not automated, we could automate it in CI (by diffing gh-pages branch with newly generated docs?)

**Comment by justinmk** [comment]
Definitely. The manual steps are listed here (but there's likely a much better way, I just didn't search for it...): https://github.com/neovim/node-client?tab=readme-ov-file#regenerate-documentation-website
